index.bs

```diff
@@ -5881,15 +5881,15 @@ To <dfn data-lt="unidirectionally broadcasting the shapes">unidirectionally broa
 
 1. Let |sizeA| be |shapeA|'s [=list/size=].
 1. Let |sizeB| be |shapeB|'s [=list/size=].
-1. If |sizeB| > |sizeA|, then return failure.
-1. Let |paddedB| be a [=list/clone=] of |shapeB|.
-1. While |paddedB|'s [=list/size=] is less than |sizeA|, [=list/prepend=] 1 to |paddedB|.
+1. If |sizeA| > |sizeB|, then return failure.
+1. Let |paddedA| be a [=list/clone=] of |shapeA|.
+1. While |paddedA|'s [=list/size=] is less than |sizeB|, [=list/prepend=] 1 to |paddedA|.
 1. Let |outputShape| be a new [=/list=].
-1. [=list/For each=] |index| in [=the range=] 0 to |sizeA|, exclusive:
-    1. Let |dimA| be |shapeA|[|index|].
-    1. Let |dimB| be |paddedB|[|index|].
+1. [=list/For each=] |index| in [=the range=] 0 to |sizeB|, exclusive:
+    1. Let |dimA| be |paddedA|[|index|].
+    1. Let |dimB| be |shapeB|[|index|].
     1. If |dimA| is not equal to |dimB| and |dimA| is not equal to 1, then return failure.
```

Comment:
I want to defer to @fdwr but call out:

If the claim _"[The current definition is correct given the interpretation of ...](https://github.com/webmachinelearning/webnn/pull/663#issuecomment-2081749493)"_ is correct, and this change is turning into a strict (1) reorder and (2) rename of the params, then it seems like this line needs to be altered to maintain the current behavior. 

Also, @fdwr's note about updating the "call sites" needs to happen as well?